test/test_approximate_shortest_superstring.py

```diff
@@ -0,0 +1,74 @@
+import unittest
+import os
+import math
+import random
+
+from approx_shortest_superstring.approximate_shortest_superstring \
```

Comment:
Nit: when using this type of linebreaks please add some spaces on the second line - it does not change anything with automatic styleguides, but I guess it's more readable than when both start equally.

Reply:
I don't understand, can You please give me an example of such solution?

Reply:
I was thinking of something like:
from something_very_very_long \
    import something

Don't worry, I'll correct it on my own later during merges of all your codes.